resultbrowser/src/css/default.css

```diff
@@ -627,3 +632,242 @@ table.key-value-table td.value .csep {
         transform: rotate(360deg);
     }
 }
+
+.actionimgScale {
+	position: absolute;
+    z-index: 10;
+    width: 50px;
+    height: 50px;
+    background: rgba(255, 255, 255, 0.5);
+    align-content: center;
+    text-align: center;
+    border-radius: 5px;
+    border: 1px solid #bbb;
+    cursor: pointer;
+}
+
+.actionimgScale#actionimgSwitch{
+	right: 20px;
+	bottom: 20px;
+}
+
+.actionimgScale#actionimgSwitch.full {
+	font-size: 30px;
+}
+
+.actionimgScale#actionimgSwitch.fit {
+	font-size: 20px;
+}
+
+/********************************************************************************************
+ *
+ * Responsive
+ *
+ ********************************************************************************************/
+
+#wrapper, #mainContent {
+	flex-flow: row;
+	width: 100vw !important;
+}
+
+#header .expander {
+	display: none;
+}
+
+#menu-close {
+	display: none;
```

Comment:
How about adding `cursor: pointer;`?

Reply:
To which element? `#menu-close` is only ever visible in mobile mode, where you don't see a cursor.

Reply:
Haha, true!